src/test/java/StepDefinitions/Steps.java

```diff
@@ -2,53 +2,117 @@
 import io.cucumber.java.en.And;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.openqa.selenium.support.ui.Select;
 import org.testng.Assert;
-
 import java.util.concurrent.TimeUnit;
 
-
 public class Steps {
     public String baseUrl = "https://orswebsitedev.azurewebsites.net/ors";
     public String homeUrl = "https://orswebsitedev.azurewebsites.net/";
+    public String createReviewUrl = "https://orswebsitedev.azurewebsites.net/ors/create-item/Review";
     public WebDriver driver;
 
+
     @Given("the user is on the login page")
-    public void the_user_is_on_the_login_page() {
-        //Use the setProperty Java method to set the chrome driver path
-        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver");
-        driver = new ChromeDriver();
-        driver.get(baseUrl);
-    }
-    @When("the user enters a valid {string} and a valid {string}")
-    public void the_user_enters_a_valid_and_a_valid(String arg1, String arg2) {
-        driver.findElement(By.name("email")).sendKeys(arg1);
-        driver.findElement(By.name("password")).sendKeys(arg2);
-    }
+        public void login_page() {
+            //Use the setProperty Java method to set the chrome driver path
+            System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver");
+            driver = new ChromeDriver();
```

Comment:
Properties and initialization for the ChromeDriver should be external in a central place. You might want to e.g. change the properties it is initialized with, or maybe even switch to a different driver - so worth just using the WebDriver base class if at all possible.